### ISSUE TITLE
Feature: Display char counter when editing profile bio

### ISSUE THREAD

**Issue by realpixelcode** [mastodon/mastodon-android#242]
Since Mastodon enforces a limit of characters in the profile bio, it would be extremely useful to have an explicit characters counter when editing it. 

If your bio is too long, you get an error message that only says “more than 500 chars”, but it doesn't actually tell you exactly how many you used.

**Comment by Zhormos** [comment]
Yes I need this too. There is already such a counter in the post editor, may it be possible to re-use it in the bio editor?  I've come up with a tip: i'll write my bio in the post editor and then paste it in my bio.
